core/src/main/java/org/akaza/openclinica/web/filter/ApiSecurityFilter.java

```diff
@@ -239,8 +249,8 @@ public OCUserDTO getUserDetails (HttpServletRequest request) {
     }
 
     //TODO Put this somewhere else?
-    private Map<String, String>  createRandomizeUserAccount() throws Exception {
-        Map<String, String> map = new HashMap<>();
+    private HashMap<String, String>  createRandomizeUserAccount() {
```

Comment:
Not sure why Map was replaced by HashMap. Good practice to use interfaces whenever possible.